### PR TITLE
Fix nothing not in EmptySpace.

### PR DESCRIPTION
**Pull request by aterenin**

### CODE REVIEW

File: src/implementations/spaces/empty_space.jl [hunk 1]
```diff
@@ -4,5 +4,5 @@ struct EmptySpace <: AbstractSpace end
 
 Base.eltype(s::EmptySpace) = nothing
```
Comment (findmyway) [inline src/implementations/spaces/empty_space.jl]: `nothing` -> `Nothing`?

Comment (aterenin) [reply src/implementations/spaces/empty_space.jl]: Thanks!

**Comment by findmyway** [comment]
The format-check can be ignored. Usually, an automatic PR will be made to fix that.

**Comment by findmyway** [comment]
@JuliaRegistrator register

**Pull request merged**